src/gameserver/components/embed/applet/applet.vue

```diff
@@ -18,13 +18,16 @@
 				<translate>You may use the button below to get it.</translate>
 			</p>
 
-			<a href="https://java.com/download" onclick="window.open( this.href ); return false">
+			<external-link
+				href="https://java.com/download"
+				onclick="window.open( this.href ); return false"
```

Comment:
i know its unrelated, but shouldnt we just do target blank?

Reply:
This should be a prop on external-link, imo. <external-link popup> or something stupid